### PR TITLE
Closes #241 removes unused packages

### LINKED ISSUE #241: Remove unused dependancies
### Feature Idea

remove the janitor and tm packages from the dependancy it seems like you only use these packages in the dev folder not the r folder

### Relevant Input

_No response_

### Relevant Output

_No response_

### Reproducible Example/Pseudo Code

_No response_

### PR DESCRIPTION
**Pull request by bms63**
@atorus-research/xportr-development-team hi all.  I did a quick release for xportr to remove the janitor and tm packages from imports as requested by GSK.  This was to help get the package into GSK's frozen environment.  

This PR just catches up the recent update

### CODE REVIEW

**Comment by bms63** [comment]
@elimillera and @EeethB can one of you just sanity check this please so I can merge in.

File: DESCRIPTION [hunk 1]
```diff
@@ -28,17 +28,15 @@ Imports:
     checkmate,
     cli,
     dplyr (>= 1.0.2),
-    glue (>= 1.4.2),
     haven (>= 2.5.0),
-    janitor,
```
Comment (vedhav) [inline DESCRIPTION:33]: We still rely on using `janitor::make_clean_names` and `tm::stemDocument()` inside the `xportr_tidy_rename()`. So, I think we cannot remove them yet.

File: NEWS.md [hunk 2]
```diff
@@ -25,6 +30,8 @@
 
 * New argument in `xportr_length()` allows selection between the length from metadata, as previously done, or from the calculated maximum length per variable when `length_source` is set to “data” (#91)
 
+* * Make `xportr_type()` drop factor levels when coercing variables
```
Comment (vedhav) [inline NEWS.md:33]:
```suggestion
* Make `xportr_type()` drop factor levels when coercing variables
```

Just a typo

File: DESCRIPTION (hunk 1 above)
Comment (bms63) [reply DESCRIPTION:33]: I did the same thing.  The function is in `dev` folder and not used in xportr.

Comment (vedhav) [reply DESCRIPTION:33]: 🙈 Sorry didn't realize the search hit was inside the `dev` folder.

Comment (bms63) [reply DESCRIPTION:33]: I said the same thing to the person at GSK reviewing the content lol

**Pull request merged**